### DISCUSSION TITLE
How can I set tag to note?

### DISCUSSION THREAD

**Discussion by korney4eg** [glushchenko/fsnotes#1297] [Q&A] [answered]
Hey guys,

Thanks for great application. I bought it for iOS and MacOS after hearing it in radio-t.
I couldn't find where I set/update tags for the note. Can you help me?

**Comment by glushchenko** [comment, marked as answer]
Hi, Alexey! Just type #tag inside text editor in macOS, or press second toolbar item in iOS.

**Reply by korney4eg** [reply]
Thanks @glushchenko .

I got confused and mixed up headers `# something` with tags `#onotherthing`. So to sum up, you just need to place `#tagname` in any place inside note.
